lcidlc/lclink.sh

```diff
@@ -131,7 +132,7 @@ else
 
 		# Build the 'dylib' form of the external - this is used by simulator builds, and as
 		# a dependency check for device builds.
-		"$DEVELOPER_BIN_DIR/g++" -stdlib=libc++ -dynamiclib -arch $ARCH -miphoneos-version-min=${MIN_VERSION} -isysroot "$SDKROOT" $FRAMEWORK_SEARCH_PATHS $STATIC_FRAMEWORKS -o "$DYLIB_FILE" "$BUILT_PRODUCTS_DIR/$EXECUTABLE_NAME" $SYMBOL_ARGS $SYMBOLS $DEPS $OTHER_FLAGS
+    "$DEVELOPER_BIN_DIR/g++" -stdlib=libc++ -dynamiclib -arch $ARCH -miphoneos-version-min=${MIN_VERSION} -isysroot "$SDKROOT" $FRAMEWORK_SEARCH_PATHS $STATIC_FRAMEWORKS -o "$DYLIB_FILE" "$BUILT_PRODUCTS_DIR/$EXECUTABLE_NAME" $SYMBOL_ARGS $SYMBOLS $DEPS $OTHER_FLAGS
```

Comment:
A minor style suggestion: it can often be easier to read and edit very long command lines like this if you split them over multiple lines using `\` line continuation characters, e.g.

```bash
"$DEVELOPER_BIN_DIR/g++" -stdlib=libc++ -dynamiclib \
    -arch $ARCH -miphoneos-version-min=${MIN_VERSION} \
    -isysroot "$SDKROOT" \
    $FRAMEWORK_SEARCH_PATHS $STATIC_FRAMEWORKS \
    -o "$DYLIB_FILE" \
    "$BUILT_PRODUCTS_DIR/$EXECUTABLE_NAME" \
    $SYMBOL_ARGS $SYMBOLS \
    $DEPS $OTHER_FLAGS
```

It can also make it easier to see what changed in the command when viewing diffs.